src/index.html

```diff
@@ -1,13 +1,307 @@
+            </div>
+          </div>
+          <div class="swiper-slide swiper-slide-col-4 swiper-slide-row-2">
+            <div class="swiper-imagewrap swiper-imagewrap-row-2">
+              <img
+                src="./images/gallery/gallery_4.jpg"
+                alt="sea"
+                class="swiper-image"
+              >
+            </div>
+          </div>
+        </div>
+        <div class="swiper-pagination gallery__pagination"></div>
+      </div>
+    </section>
+
+    <section id="subscription" class="subscription">
+      <h2 class="page__title subscription__title">Підписка</h2>
+      <p class="subscription__text">Бути в курсі всіх актуальних подій та останніх новин.</p>
+      <form action="" class="form subscription__email">
+        <div class="form__text">
+          <input type="email" name="email" placeholder="E-mail" class="form__input" required>
+        </div>
+        <button class="form__arrow" type="submit">
+          <img src="./images/icons/arrow-black.svg" alt="arrow" class="form__image">
+        </button>
+      </form>
+    </section>
+
+    <section id="contacts" class="contacts">
+      <div class="contacts__top">
+        <a href="#">
+          <img src="./images/icons/logo-contact.svg" alt="logo" class="contacts__logo">
+        </a>
+        <div class="contacts__social">
+          <a href="https://www.facebook.com/namu.museum" target="_blank">
+            <img src="./images/icons/facebook.svg" alt="facebook">
+          </a>
+          <a href="https://www.instagram.com/namu.museum/" target="_blank">
+            <img src="./images/icons/instagram.svg" alt="instagram">
+          </a>
+        </div>
+      </div>
+
+      <div class="contacts__wrapper">
+        <div class="contacts__grafic grafic">
+          <p class="grafic__title">
+            Графік роботи
+          </p>
+
+          <div class="grafic__week grafic__week--contacts">
+            <p><span class="grafic__bold">12:00 — 20:00</span> середа</p>
+            <p><span class="grafic__bold">12:00 — 20:00</span> четвер</p>
+            <p><span class="grafic__bold">12:00 — 20:00</span> п’ятниця</p>
+            <p><span class="grafic__bold">11:00 — 19:00</span> субота</p>
+            <p><span class="grafic__bold">11:00 — 19:00</span> неділя</p>
+          </div>
+
+          <div class="grafic__kasa">
+            <p><span class="grafic__bold">Вихідні:</span> понеділок, вівторок
+              <br>Каса припиняє роботу за 1 годину до закриття музею
+            </p>
+          </div>
+        </div>
+
+        <div class="contacts__contact">
+          <p class="contacts__text">
+            Контакти:
+          </p>
+          <a href="https://goo.gl/maps/op682XHSmBEYuNpn7" class="contacts__adress" target="_blank">
+            вул. М. Грушевського, 6; Київ, 01001
+          </a>
+          </br>
+          <span class="contacts__adress">тел. </span>
+          <a href="tel: +38 (044) 278-13-57" class="contacts__adress">
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
          <a href="tel:+380442781357" class="contacts__adress">
```